node/src/components/rest_server.rs

```diff
@@ -210,6 +210,8 @@ mod tests {
         let actual_schema = serde_json::to_string_pretty(&actual_schema).unwrap();
         let actual_schema: Value = serde_json::from_str(&actual_schema).unwrap();
 
+        // println!("{}", actual_schema);
```

Comment:
Do we need this?

Reply:
Yes, its being used for when changes are made to the RPC schema